exercises/list-ops/example.rb

```diff
@@ -0,0 +1,67 @@
+class ListOps
+  VERSION = 1
+  ## Do not use the functions map, concat, count, length, etc.
+  ## and other built in ruby methods to solve  these
+  ##
+  def self.arrays(arr)
+    counter = 0
+    arr.each do
+      counter += 1
+    end
+    counter
+  end
+
+  def self.reverser(arr)
+    answer = []
+    until arr.empty?
+      answer << arr.pop
+    end
+    answer
+  end
+
+  def self.concatter(arr1, arr2)
+    answer = []
+    arr1.each do |element|
+      answer << element
+    end
+    arr2.each do |element|
+      answer << element
+    end
+    answer
+  end
+
+  def self.mapper(arr)
+    answer = []
+    arr.each do |element|
+      answer << (element + 1)
+    end
+    answer
+  end
+
+  def self.filterer(arr)
```

Comment:
I think methods like this could receive a block without relying on a fixed condition like `element % 2 == 1`. It would be something like `self.filter(arr, &block)`. With this you can call your block inside the `each` method with `block.call(element)`.